frontend/src/metabase-types/api/mocks/database.ts

```diff
@@ -12,6 +12,10 @@ export const COMMON_DATABASE_FEATURES: DatabaseFeature[] = [
   "nested-queries",
   "standard-deviation-aggregations",
   "persist-models",
+  "left-join",
+  "right-join",
+  "inner-join",
+  "full-join",
```

Comment:
Turns out we didn't have database `join` features typed anywhere. This PR adds them and includes them in the default feature list for mock databases